docs/modules/client-failover/examples/ClientWithSsl.java

```diff
@@ -113,6 +113,7 @@ private static void nonStopMapExample(HazelcastInstance client) {
                 e.printStackTrace();
             }
             // end::catch-exceptions[]
+            // map.put("key-" + randomKey, "value-" + randomKey); Replaced by exception handling
```

Comment:
This should not be commented out because we say 

```
If you’re using the nonStopMapExample in the sample Java client, your client stops. This is expected because write operations are not retryable when a cluster is disconnected. The client has sent a put request to the cluster but has not received a response, and so the result of the request is unknown. To prevent the client from overwriting more recent write operations, this write operation is stopped and an exception is thrown.
```

in the last paragraph of Step 3.

Reply:
The only problem with uncommenting this line in the sample code, is that the same code is shared by the example (https://github.com/hazelcast-guides/client-failover/blob/DOCS-271_Missing_Code_From_Code_Sample/docs/modules/client-failover/examples/ClientWithSsl.java). If this line is introduced then the code wouldn't run.

Reply:
I did not understand. You are saying that this code is shared by `ClientWithSsl.java`, but it is the same file as this file. Am I missing something ?